### PR TITLE
Feat/better intro

### PR DESCRIPTION
**Pull request by andrewlew1s**
Currently:

Screen Shot 2020-03-15 at 11 54 31 PM

### CODE REVIEW

File: website/src/Website.vue [hunk 1]
```diff
@@ -4,7 +4,10 @@
 		<section class="Website__content">
 
 			<p>
-				Welcome to Holistic Toolbox. We provide a growing list of tools to make working easier.
+				We're a team of developers striving to create a useful set of tools for developers, computer scientists and curious folk.
```
Comment (michaelfitzhavey) [inline website/src/Website.vue]: Can you add a screenshot of what this looks like? As mentioned on the call I think potentially one paragraph of content may look nicer than using lots o space on the page for just two sentences.

**Comment by michaelfitzhavey** [comment]
This is part of https://github.com/holistic-web/toolbox/issues/92

File: website/src/Website.vue (hunk 1 above)
Comment (andrewlew1s) [reply website/src/Website.vue]: p > 2p

Comment (michaelfitzhavey) [reply website/src/Website.vue]: My reasoning is think it's nice to keep as much space on the page for the list of tools as possible. It looks like an improvement to me now!

**Pull request merged**